README.md

```diff
@@ -1,6 +1,16 @@
 # Prism
 
-Prism is a Samply component ran centrally, which queries sites for criteria and number of results for these criteria in their Samply.Blaze stores. Prism is to be used by Samply.Spot or Samply.Lens for showing expected number of results next to criteria in the search tree. 
+Prism is a Samply component ran centrally, which queries sites for criteria and number of results for these criteria in their [Samply.Blaze](https://github.com/samply/blaze) stores. Prism is to be used by [Samply.Spot](https://github.com/samply/spot/) or [Samply.Lens](https://github.com/samply/lens) for showing expected number of results next to criteria in the search tree. 
+
+## How it works
+
+Samply.Prism returns cumulative positive numbers of each of the individual criteria, defined in CQL queries and Measures, at sites it is queried about. Those numbers correspond to the expected number of results when query with an individual criterion is issued.
+
+Prism does not return all the possible search criteria in the search tree and is not a replacement for a catalogue, instead its results are to be injected into an existing catalogue. It doesn't return criteria for which there are no results in at least one store. It does not return results for range types, except patient age, stratified by years (in which case it is up to the frontend to add up all the results within the interval whenever the interval is changed by the user).
```

Comment:
> in which case it is up to the frontend to add up all the results within the interval whenever the interval is changed by the user

I would suggest to remove this part in parentheses as this is not currently implemented in Lens nor planned